### PR TITLE
chore: [CDS-73199]: add serverless 2.0 specific step icons

### PR DESCRIPTION
**Pull request by chetanvekariya-harness**
You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

File: packages/icons/src/serverless-aws-lambda-package-inverse.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg width="33" height="32" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M5.727 5.5a.5.5 0 0 1 .5-.5h7.875a.5.5 0 0 1 .416.223l2.476 3.715h10.233a.5.5 0 0 1 .5.5v10.141a.5.5 0 1 1-1 0V9.938h-7.212v1.701h1.157a.5.5 0 0 1 0 1h-1.157v1.275h1.157a.5.5 0 1 1 0 1h-1.157v1.099h1.157a.5.5 0 0 1 0 1h-1.157v1.276a.5.5 0 1 1-1 0v-1.276h-1.158a.5.5 0 1 1 0-1h1.158v-1.1h-1.158a.5.5 0 1 1 0-1h1.158V12.64h-1.158a.5.5 0 1 1 0-1h1.158V9.938h-1.788a.5.5 0 0 1-.416-.223L13.834 6H6.727v20h13.392a.5.5 0 0 1 0 1H6.227a.5.5 0 0 1-.5-.5v-21z" fill="#fff"/><path d="m23.773 32 6.01-5.443-.447-.28-5.094 4.61v-1.09l5.658-5.119v1.099l-.564.5.447.28.602-.535v-2.123l-1.318.182 1.181-1.978v-1.866L29.986 20l-6.918 5.96v1.46l2.22-.375-1.516 2.563V32zm5.991-10.81v.768l-1.268 2.201-.645.09 1.913-3.06zm-6.23 5.646v-.788l2.7-.368-.567.818-2.134.338z" fill="#fff"/></svg>
```
Comment (AshwinBhatkal-Harness) [inline packages/icons/src/serverless-aws-lambda-package-inverse.svg:1]: Let's remove width and height and use viewBox instead

File: packages/icons/src/serverless-aws-lambda-deploy-v2-inverse.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 33 32" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M7.296 3.785a.5.5 0 0 1 .683-.183l4.812 2.778a.5.5 0 0 1-.5.866L7.48 4.468a.5.5 0 0 1-.183-.683zm-2.87 2.596a.5.5 0 1 0-.5.866l4.812 2.778a.5.5 0 0 0 .5-.866L4.426 6.38zm13.594.51a.5.5 0 0 0-.5-.001l-7 4.04a.5.5 0 0 0-.25.433v7.785a.5.5 0 0 0 .248.432l7.329 4.264a.5.5 0 0 0 .502.001l7.031-4.07a.5.5 0 0 0 .25-.433v-7.749a.5.5 0 0 0-.25-.432l-7.36-4.27zM11.27 18.86v-6.63l6.38 3.69-.046 6.626-6.335-3.686zm7.334 3.682.046-6.62 5.98-3.462v6.594l-6.026 3.488zm-.452-7.487 5.98-3.462L17.77 7.9l-6 3.463 6.383 3.69zm-13.726-2.63a.5.5 0 0 0-.5.866l4.812 2.778a.5.5 0 1 0 .5-.866l-4.812-2.778z" fill="#fff"/><path d="m23.773 32 6.01-5.443-.447-.28-5.094 4.61v-1.09l5.658-5.119v1.099l-.564.5.447.28.602-.535v-2.123l-1.318.182 1.181-1.978v-1.866L29.986 20l-6.918 5.96v1.46l2.22-.375-1.516 2.563V32zm5.991-10.81v.768l-1.268 2.201-.645.09 1.913-3.06zm-6.23 5.646v-.788l2.7-.368-.567.818-2.134.338z" fill="#fff"/></svg>
```
[the remark this answers is not in the document]
Comment (AshwinBhatkal-Harness) [reply packages/icons/src/serverless-aws-lambda-deploy-v2-inverse.svg:1]: Same goes for other inverse icons

File: packages/icons/src/serverless-aws-lambda-package-inverse.svg (hunk 1 above)
Comment (chetanvekariya-harness) [reply packages/icons/src/serverless-aws-lambda-package-inverse.svg]: Done.

File: packages/icons/src/serverless-aws-lambda-deploy-v2-inverse.svg (hunk 2 above)
[the remark this answers is not in the document]
Comment (chetanvekariya-harness) [reply packages/icons/src/serverless-aws-lambda-deploy-v2-inverse.svg]: Done.

**Review by samarth-sharma31 — APPROVED** [review]

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Review by samarth-sharma31 — APPROVED** [review]

**Pull request merged**